encrypted-media-respec.html

```diff
@@ -3179,44 +3193,66 @@
+                Implementations with such support SHOULD expose the ability for the user to select this option.
               </p>
+              <div class="note">
+                <p>
+                  When supported, applications can select for this mode using <a def-id="option-distinctiveIdentifier"></a> = <a def-id="requirement-not-allowed"></a>.
+                  Selecting such an option MAY affect the results of the <a def-id="requestMediaKeySystemAccess"></a> call and/or the license requests that are generated from subsequently generated sessions.
+                </p>
+                <p>
+                  Providing the user access to select or choose this implementation capability may allow the user to access content while maintaining a higher degree of privacy.
+                </p>
+              </div>
             </li>
           </ul>
         </section>
 
         <section id="encrypt-identifiers">
           <h3>Encrypt Identifiers</h3>
-          <p>When exposed to the application - either via a <a def-id="message"></a> event or a message from the server, such as one that is passed to <a def-id="update"></a> - <a def-id="distinctive-identifiers"></a> MUST be encrypted at the message exchange level.
+          <p>
+            <a def-id="distinctive-identifiers"></a> and <a def-id="distinctive-permanent-identifiers"></a> MUST be encrypted at the message exchange level when exposed outside the client.
+            All other identifiers SHOULD be encrypted at the message exchange level when exposed outside the client.
             The encryption MUST ensure that the ciphertext cannot be used as a proxy for the actual identifier, even given the same plaintext.
-            The CDM MUST verify that the encryption key belongs to a valid license server for its Key System.
           </p>
           <p class="issue"><a href="https://github.com/w3c/encrypted-media/issues/219">Issue 219</a> - Add more specific text about ensuring the desired privacy properties when encrypting identifiers.</p>
-          <p>This MAY be implemented using a <a href="#server-certificate">server certificate</a>.</p>
-          <p>The license server MUST NOT expose a <a def-id="distinctive-identifier"></a> to any entity other than the CDM that sent it.</p>
-          <a def-id="distinctive-permanent-identifiers"></a> MUST also be encrypted at the message exchange level.
+          <p class="note">Identifiers may be exposed in the following ways:</p>
+            <ul>
+              <li><p>To the application via a <a def-id="message"></a> event.</p></li>
+              <li><p>In a message from a server, such as one that is passed to <a def-id="update"></a>.</p></li>
+              <li><p>As part of <a href="#privacy-individualization">individualization</a>.</p></li>
+            </ul>
```

Comment:
I think you intended the list to appear in the note as well, but it [does not](https://rawgit.com/ddorwin/encrypted-media/259b372a13888b2b52f9364bbe0f096fb3d8c4ad/encrypted-media-respec.html#encrypt-identifiers).